### PR TITLE
Disable tests that rely on GetComObjectData() and SetComObjectData() …

### PR DESCRIPTION
**Pull request by AaronRobinsonMSFT**
…semantics

Fix bug in Prelink_NonRuntimeMethod_ThrowsArgumentException test.

see #31188, #31153

### CODE REVIEW

**Comment by AaronRobinsonMSFT** [comment]
cc @hughbe

File: src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/PrelinkTests.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ public void Prelink_NullMethod_ThrowsArgumentNullException()
         [Fact]
         public void Prelink_NonRuntimeMethod_ThrowsArgumentException()
         {
-            AssertExtensions.Throws<ArgumentException>(null, () => Marshal.Prelink(new NonRuntimeMethodInfo()));
+            AssertExtensions.Throws<ArgumentException>("m", () => Marshal.Prelink(new NonRuntimeMethodInfo()));
```
Comment (stephentoub) [inline src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/PrelinkTests.cs]: Is this going to pass CI without the coreclr update that's coming in?

Comment (AaronRobinsonMSFT) [reply src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/PrelinkTests.cs]: @stephentoub Nope. This is going to fail. I should add the `ActiveIssue`

**Comment by AaronRobinsonMSFT** [comment]
I am going to abandon this PR since I have updated the official insertion PR (#31188 ) with the needed changes.

**Pull request closed without merging**